operatorhub/openshift/config.yaml

```diff
@@ -56,6 +56,13 @@ image-substitutions:
       containerName: openshift-pipelines-operator
       envKeys:
       - IMAGE_PIPELINES_ARG__KUBECONFIG_WRITER_IMAGE
+- image: registry.redhat.io/openshift-pipelines/pipelines-workingdirinit-rhel8@
```

Comment:
Could we re-use an existing image ?

Reply:
we can, if we decide so.
Let us merge this as this follows the default expectation.

Then if we decide to use an existing image, we can change the image name part. This entry  will server as a placeholder

Reply:
sounds good 👍🏼